internal/network/command.go

```diff
@@ -126,13 +120,13 @@ func printTable(cmd *cobra.Command, network networkingv1.NetworkingV1Network) er
 		}
 
 		switch cloud {
-		case CloudAws:
+		case resource.CloudAws:
```

Comment:
I'm not sure I would call these resources? I could be wrong but I thought resources were something that users created. This might be better suited for the `cloud` package, where this would be called `cloud.Aws`.